src/PostgREST/Error.hs

```diff
@@ -259,13 +259,14 @@ instance JSON.ToJSON SQL.CommandError where
     "details" .= (fmap T.decodeUtf8 d :: Maybe Text),
     "hint"    .= JSON.Null]
 
+
 pgErrorStatus :: Bool -> SQL.UsageError -> HTTP.Status
 pgErrorStatus _      (SQL.ConnectionUsageError _) = HTTP.status503
-pgErrorStatus _      SQL.PoolIsReleasedUsageError = HTTP.status500
+pgErrorStatus _      SQL.AcquisitionTimeoutUsageError = HTTP.status504
 pgErrorStatus _      (SQL.SessionUsageError (SQL.QueryError _ _ (SQL.ClientError _)))      = HTTP.status503
 pgErrorStatus authed (SQL.SessionUsageError (SQL.QueryError _ _ (SQL.ResultError rError))) =
   case rError of
-    (SQL.ServerError c m _ _) ->
+    (SQL.ServerError c m _ _ _) ->
```

Comment:
Ok so we don't need this parameter as it looks more fit for detecting an error when sending raw SQL queries.

https://hackage.haskell.org/package/hasql-1.6.0.1/docs/Hasql-Session.html#t:ResultError

([Maybe](https://hackage.haskell.org/package/base-4.14.1.0/docs/Data-Maybe.html#t:Maybe) [Int](https://hackage.haskell.org/package/base-4.14.1.0/docs/Data-Int.html#t:Int))	
Position. Error cursor position as an index into the original statement string. Positions are measured in characters not bytes.

Reply:
Yes, I had it in first but it resulted mostly in `null` values in the tests, and where it did give a number it didn't seem particularly useful information, should have mentioned that.